echo_server.rb

```diff
@@ -0,0 +1,47 @@
+class UserEcho
+
+	def initialize(welcome, instructions)
+		@welcome = "Welcome!"
+		@instructions = "Please enter a word to be reprinted, to exit this program please enter exit:"
+	end
+
```

Comment:
Perhaps it'll be easier to read if you group related methods together.

welcome_message
instruction_message
goodbye_message